### ISSUE TITLE
[BUG] advantages is incorrectly passed as response_mask to policy_loss_fn in dp_actor.py

### ISSUE THREAD

**Issue by liujunzhuo** [volcengine/verl#2506]
https://github.com/volcengine/verl/blob/8e0b9bd9e506937bde2216a0ac41907ab6aaad83/verl/workers/actor/dp_actor.py#L515

A bug was identified in the policy training step where the policy loss function is **called with an incorrect parameter.** The response_mask argument is being passed the advantages instead of the actual response mask.

```python
policy_loss_fn = get_policy_loss_fn(loss_mode)
pg_loss, pg_clipfrac, ppo_kl, pg_clipfrac_lower = policy_loss_fn(
    old_log_prob=old_log_prob,
    log_prob=log_prob,
    advantages=advantages,
    response_mask=advantages,  # <--- BUG: Should be the actual response_mask
    loss_agg_mode=loss_agg_mode,
    config=self.config,
)
```

**Comment by liujunzhuo** [comment]
Thanks for your quick fix! I'm really grateful for the Verl framework.
